static/panes/editor.ts

```diff
@@ -807,18 +805,17 @@ export class Editor extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Edit
             compilers.forEach(compiler => {
                 let knownCompiler = false;
 
-                const compilerExtInfo = this.hub.compilerService.findCompiler(
-                    this.currentLanguage?.id ?? '',
-                    compiler.compiler
+                const compilerExtInfo = unwrap(
+                    this.hub.compilerService.findCompiler(this.currentLanguage?.id ?? '', compiler.compiler)
                 );
                 const semver = this.cleanupSemVer(compilerExtInfo.semver);
                 let groupOrName = compilerExtInfo.baseName || compilerExtInfo.groupName || compilerExtInfo.name;
                 if (semver && groupOrName) {
                     groupOrName = groupOrName.toLowerCase();
-                    if (groupOrName.indexOf('gcc') !== -1) {
+                    if (groupOrName.includes('gcc')) {
```

Comment:
Heads-up: This has been stuck in indexOf for so long because we used to want ES5 support in the frontend, and while https://caniuse.com/?search=include said it was a non-issue, we have some reports of things breaking for some people with similar code at some point. I say let's go ahead with this and if someone complains again, we can always revert, but it's 2023 already! About time to forget about these kind of issues!

Reply:
Amen. And good spot @RubenRBS

Reply:
Will our webpack / the ts compiler compile the `.includes` into something even more widely supported? I know that's possible in theory, not sure how we have things setup.